test/libyul/yulOptimizerTests/fullSuite/extcodelength.yul

```diff
@@ -0,0 +1,42 @@
+{
+    let _1 := 0
+    let value := calldataload(4)
+    if iszero(eq(value, and(value, sub(shl(160, 1), 1)))) { revert(_1, _1) }
+    let length := extcodesize(value)
+    let _2 := 0xffffffffffffffff
+    if gt(length, _2) { revert(0, 0) }
+    let _3 := not(31)
+    let memPtr := mload(64)
+    let newFreePtr := add(memPtr, and(add(and(add(length, 31), _3), 63), _3))
+    if or(gt(newFreePtr, _2), lt(newFreePtr, memPtr)) { revert(0, 0) }
+    mstore(64, newFreePtr)
+    mstore(memPtr, length)
+
+    // We aim to optimize this out.
+    // TODO it does not work :(
```

Comment:
Would be nice to fix this, but maybe the arbitrary length makes this impossible.

Reply:
If the return below is commented out, then the extcodecopy gets removed.